src/Analysis/Core/Impl/Interpreter/PythonLibraryPath.cs

```diff
@@ -28,41 +29,80 @@
 using IOPath = System.IO.Path;
 
 namespace Microsoft.Python.Analysis.Core.Interpreter {
-    public sealed class PythonLibraryPath {
-        private readonly string _modulePrefix;
+    public enum PythonLibraryPathType {
+        Unspecified,
+        StdLib,
+        Site,
+        Pth,
+    }
 
+    public sealed class PythonLibraryPath : IEquatable<PythonLibraryPath> {
         private static readonly Regex ParseRegex = new Regex(
-            @"(?<path>[^|]+)\|(?<stdlib>stdlib)?\|(?<prefix>[^|]+)?"
+            @"(?<path>[^|]+)\|(?<type>[^|]+)\|(?<prefix>[^|]+)?"
```

Comment:
What this Regex is for?

Reply:
It splits out per-line output of the script. Arguably, it should just be a split on `|`, which I can do; I just did a minimal change to what was here.